lib/base-controller.js

```diff
@@ -3,10 +3,12 @@
 var Controller = require('hmpo-form-wizard').Controller;
 var util = require('util');
 var _ = require('lodash');
+var mixins = require('./middleware/mixins');
 
 var BaseController = function BaseController(options) {
   this.confirmStep = options.confirmStep || '/confirm';
   Controller.apply(this, arguments);
+  this.use(mixins);
```

Comment:
Also it needs to attach itself to res.locals to be used in template (with current setup) so needs to either be a middleware or put in the controller somewhere (which would have the same effect on request/response)